content/efcore/getting-started/samples/defining-unique-identifiers/composite-key/README.md

```diff
@@ -0,0 +1,8 @@
+# Composite Key
+
+This code example demonstrates how to configure a composite key in EF Core via Fluent API.
+
+## Code Files:
+- [Program.cs](Program.cs)
+- [LibraryContext.cs](LibraryContext.cs)
+- [CheckoutRecord.cs](CheckoutRecord.cs)
```

Comment:
This example should still have `Book.cs` and `Reader.cs`. Just use implementations that don't require `Address.cs` and `Author.cs`.